src/components/Shared/UI/DTETypography/DTEHeader/DTEHeader.tsx

```diff
@@ -105,4 +109,18 @@ const DTEHeader = styled.h1<DTEHeaderProps>`
   }
 `;
 
+const DTEHeader: FC<DTEHeaderProps> = (props) => {
+  const { captionKey, children } = props;
+
+  return (
+    <>
+      <DTEHeaderCaption contentKey={`${captionKey}-caption`} />
+      <StyledHeader {...props}>{children}</StyledHeader>
```

Comment:
Some conditional logic here depending on whether children is defined vs `content[captionKey]` having a value.